Tails specific: do not stop memlockd on shutdown to keep files needed
by tails-kexec properly loaded in memory.

--- chroot.orig/etc/init.d/memlockd	2012-09-24 10:05:13.085049457 +0200
+++ chroot/etc/init.d/memlockd	2012-09-24 10:03:30.638108333 +0200
@@ -5,3 +5,3 @@
-# Required-Stop:     $remote_fs $syslog
+# Required-Stop:
 # Default-Start:     2 3 4 5
-# Default-Stop:      0 1 6
+# Default-Stop:
